addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1197,10 +1202,14 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         fieldEl.classList.remove('s_website_form_field_hidden_if', 'd-none');
     },
     /**
+     * @param {HTMLElement} fieldEl
      * @returns {HTMLElement} The visibility dependency of the field
      */
-    _getDependencyEl() {
-        const dependencyName = this.$target[0].dataset.visibilityDependency;
+    _getDependencyEl(fieldEl = null) {
+        if (!fieldEl) {
+            fieldEl = this.$target[0];
+        }
```

Comment:
Can't you do `_getDependencyEl(fieldEl = this.$target[0]) {` ?
Also the documentation should have the argument name between `[]` to mark it optional

Reply:
Seems to be working indeed. Found a few occurrences in the code too using `this` as default param. Changed.